### ISSUE TITLE
export MCFLY_PROMPT="❯" not working on ZSH Linux (Manjaro)

### ISSUE THREAD

**Issue by deekshithanand** [cantino/mcfly#337]
Hi there, I have the following snippet in my `.zshrc` file

```
# MCFLY configs
export MCFLY_RESULTS=20
export MCFLY_INTERFACE_VIEW=BOTTOM
export MCFLY_PROMPT="❯"
eval "$(mcfly init zsh)"


```

I installed it from pacman package manager.

Mcfly version 0.7.1

Suspecting if this feature is not available in the above version, apparently #325 by @vedkothavade was merged just around two days back. If so, can you please update the README accordingly ?

The custom prompt is not showing up, instead the default prompt $ shows up.

Here is a screenshot with other settings being in effect.

![mcfly_bug](https://user-images.githubusercontent.com/24704695/223179688-85dbb370-2a49-4f03-986b-27bedffdf24d.png)

**Comment by vedkothavade** [comment]
A new version has not been tagged with the commit yet, so to use this feature until @cantino releases a new version, use a `-git` version (not sure how Manjaro holding back packages affects this), or build from source with cargo.

**Comment by cantino** [comment]
Should be released now in v0.8.0.
